src/assets/queries/tags/add_implicit_tag_to_file.sql

```diff
@@ -0,0 +1,3 @@
+-- Add an implicit tag to a file (only if it doesn't already have it)
```

Comment:
There are other queries specifically for tags. Move those into the src/assets/queries/tags folder